src/bunchkaufman.jl

```diff
@@ -217,6 +217,12 @@ BunchKaufman{T}(B::BunchKaufman) where {T} =
     BunchKaufman(convert(Matrix{T}, B.LD), B.ipiv, B.uplo, B.symmetric, B.rook, B.info)
 Factorization{T}(B::BunchKaufman) where {T} = BunchKaufman{T}(B)
 
+AbstractMatrix(B::BunchKaufman) = B.uplo == 'U' ? B.P'B.U*B.D*B.U'B.P : B.P'B.L*B.D*B.L'B.P
+AbstractArray(B::BunchKaufman) = AbstractMatrix(B)
+Matrix(B::BunchKaufman) = Array(AbstractArray(B))
```

Comment:
Perhaps

```suggestion
Matrix(B::BunchKaufman) = convert(Matrix, AbstractArray(B))
```

to avoid another copy, when the inner call already returns a `Matrix`?

Reply:
I made the change. Also in cholesky.jl (where I copied the code from)